packages/deprecated-component-library-helpers/styles/type.scss

```diff
@@ -1,3 +1,8 @@
+// This is the only way to handle division for the strip-unit function.
+// calc() unfortunately doesn't work in this case.
+// Example: https://sass-lang.com/documentation/breaking-changes/slash-div#transition-period
+@use "sass:math";
+
```

Comment:
Seems like this is the way to solve the below as `calc()` doesn't work in the same way as `math.div`

Reply:
Super curious about learning more on this one! Is it because we're dividing two sass variables? It seems like the other divisions in these files are okay with just using calc.